packages/templates/src/raw/fitness-apps-ranked/template.json

```diff
@@ -3625,7 +3550,7 @@
           "textAlign": "initial",
           "padding": { "locked": true, "hasHiddenPadding": false },
           "type": "text",
-          "content": "<span style=\"font-weight: 500; color: #0f2c41\">Learn how to do a Squat Correctly</span>",
+          "content": "<span style=\"font-weight: 500; color: #0f2c41\">Learn how to do a squat Correctly</span>",
```

Comment:
Lowercase `correctly` too? Looks odd otherwise.

Reply:
Agree, this is still under review, with ongoing changes. Should be ready for your review by Tuesday evening.

Sorry to have raised false alarm 😅